extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/runtime/SimpleScheduler.java

```diff
@@ -141,6 +142,21 @@ public void pause() {
         }
     }
 
+    @Override
+    public void pause(String identity) {
+        Objects.requireNonNull(identity, "Cannot pause - identity is null");
+        if (identity.isEmpty()) {
+            LOGGER.warn("Cannot pause - identity is empty");
+            return;
+        }
+        for (ScheduledTask task : scheduledTasks) {
+            if (SchedulerUtils.lookUpPropertyValue(identity).equals(task.trigger.id)) {
```

Comment:
Same comment as https://github.com/quarkusio/quarkus/pull/16855/files#r622551769

Reply:
Moved it out